rpc/types_contract.go

```diff
@@ -12,31 +12,35 @@ import (
 
 type NumAsHex string
 
-type EntryPoint struct {
+type DeprecatedCairoEntryPoint struct {
 	// The offset of the entry point in the program
 	Offset NumAsHex `json:"offset"`
 	// A unique  identifier of the entry point (function) in the program
 	Selector *felt.Felt `json:"selector"`
 }
 
+type GetClassOutput interface{}
```

Comment:
I'd like it more if the name was just `ClassOutput`. The current name seems more the name of a method rather than a type.
There's no problem if you want to leave it the way it is now